frontend/src/scenes/ingestion/v1/panels/RecordingPanel.tsx

```diff
@@ -0,0 +1,42 @@
+import { SessionRecordingSettings } from 'scenes/session-recordings/settings/SessionRecordingSettings'
+import { useActions } from 'kea'
+import { ingestionLogic } from 'scenes/ingestion/v1/ingestionLogic'
+import { LemonButton, LemonDivider, Link } from '@posthog/lemon-ui'
+
+export function RecordingPanel(): JSX.Element {
+    const { setVerify } = useActions(ingestionLogic)
+
+    return (
+        <div style={{ maxWidth: 800 }}>
+            <h1 className="font-extrabold">Setup Session Recordings</h1>
+            <p className="prompt-text">
+                Session recordings allow you to see recordings of how your users are really using your product. It
+                includes powerful features like error tracking, filtering, and analytics to <b>diagnose UI issues</b>,{' '}
+                <b>improve support</b>, and <b>generally get inspired</b>.
+            </p>
+            <p>
+                No further configuration is required to immediately start capturing recordings. Learn more about
+                specific recordings settings in our{' '}
+                <Link to="https://posthog.com/manual/recordings#using-session-recording" target="_blank">
+                    docs
+                </Link>
+                .
+            </p>
+
+            <SessionRecordingSettings inModal />
```

Comment:
Personally I'm not a fan of this. I think just dropping in the Settings here is completely overwhelming for a user. We should make it as concise as possible and enticing to go forward with.

Definitely the case for a custom screen here. Also, we can't use this component anyways as we want to only set things to be enabled when they actually click to continue.